java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java

```diff
@@ -2,7 +2,7 @@
 import com.sun.jersey.api.client.ClientHandlerException; // Compliant
 import java.util.ArrayList;
 
-class A {
+class SunPackagesUsedCheck {
```

Comment:
We should rename the class to `SunPackagesUsedCheckCustom` to match the file name and avoid confusion when looking at the two test files side by side.